src/rhsmlib/services/entitlement.py

```diff
@@ -47,6 +51,9 @@ def get_pools(self, pool_subsets=None, matches=None, pool_only=None, match_insta
         if kwargs:
             raise exceptions.ValidationError(_("Unknown arguments: %s") % kwargs.keys())
 
+        if isinstance(pool_subsets, six.string_types):
+            pool_subsets = [pool_subsets]
```

Comment:
I can confirm that this just works even when more strings of `pool_subsets` is set: `$ busctl call com.redhat.RHSM1 /com/redhat/RHSM1/Entitlement com.redhat.RHSM1.Entitlement GetPools a{sv}a{sv} 1 pool_subsets as 2 consumed installed 0`